backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/creditLimit.feature

```diff
@@ -47,11 +47,11 @@ Feature: credit limit
       | endcustomer_1            | S                   |
 
     And metasfresh contains M_Inventories:
-      | Identifier | M_Warehouse_ID | MovementDate |
-      | i_1        | 540008         | 2021-04-17   |
+      | Identifier | M_Warehouse_ID | MovementDate         |
+      | i_1        | 540008         | 2021-04-16T21:00:00Z |
```

Comment:
why this? maybe add a comment..

(LE..it looks familiar..i might have made this change in some branch..but still, it should be somehow explained)